eclass/font.eclass

```diff
@@ -4,16 +4,15 @@
 # @ECLASS: font.eclass
 # @MAINTAINER:
 # fonts@gentoo.org
-# @SUPPORTED_EAPIS: 0 1 2 3 4 5 6
+# @SUPPORTED_EAPIS: 0 1 2 3 4 5 6 7
 # @BLURB: Eclass to make font installation uniform
 
 case ${EAPI:-0} in
-	0|1|2|3|4|5|6) ;;
-	*)             die "EAPI ${EAPI} is not supported by font.eclass." ;;
+	0|1|2|3|4|5|6) inherit eutils ;;
```

Comment:
Should we nuke some of these older EAPIs that are no longer used? IIRC there was only 1-2 font ebuilds with `EAPI` 0

Reply:
let's do that in one go when we have eliminated `0`